### ISSUE TITLE
duplicate label in dynatrace-operator chart deployment-webhook.yaml

### ISSUE THREAD

**Issue by xyzopowa** [Dynatrace/helm-charts#160]
Hi ,
in dynatrace-operator/chart/default/templates/Common/webhook/deployment-webhook.yaml , 
when i render the chart , i encounter a duplicate  label "dynatrace.com/operator: dynakube"
```
  labels:
    {{- include "dynatrace-operator.commonlabelswebhook" . | nindent 4 }}
    dynatrace.com/operator: dynakube
```
(dynatrace-operator.commonlabelswebhook already contain this label)
is it possible to remove L22 ?
(even f it work using helm to deploy , it break when using a helmrelease ( fluxcd )  )

**Comment by oleterno-equinor** [comment]
I have the same problem

**Comment by chrismuellner** [comment]
Fixed in #166
